docs/machine-learning/resources/tasks.md

```diff
@@ -1,42 +1,52 @@
 
 > [!NOTE]
-> ML.NET is currently in Preview. All machine learning tasks are not currently supported. To submit a request for a certain task, please open an issue on our [GitHub repository](https://github.com/dotnet/machinelearning/issues).
+> Currently, ML.NET does not support machine learning tasks with images. Support will be added in future releases. 
+
+## Binary classification
+
+A supervised machine learning task that is used to predict which of two classes (categories) an instance of data belongs to. The input of a classification algorithm is a set of labeled examples, where each label is an integer of either 0 or 1. The output of a binary classification algorithm is a classifier, which you can use to predict the class of new unlabeled instances. Examples of binary classification scenarios include:
 
-## Binary Classification
-A supervised machine learning task that is used to predict which of two classes an instance of data belongs to. The input of a classification algorithm is a set of labeled examples. Each example is represented as a feature vector, and each label is an integer of value of 0 or 1. The output of a binary classification algorithm is a classifier, which can be used to predict the label of new unlabeled instances. Examples of binary classification scenarios include:
+* [Understanding sentiment of Twitter comments](../tutorials/sentiment-analysis.md) as either "positive" or "negative".
+* Diagnosing whether a medical patient has a certain disease or not.
+* Making a decision to send an email to a "spam" folder or not.
 
-* Understanding sentiment of Twitter comments as either "positive" or "negative"
-* Diagnosing whether a medical patient has a certain disease or not
-* Making a decision to send an email to a "spam" folder or not
+## Multi-class classification
 
-## Multi-class Classification
-A supervised machine learning task that is used to predict the category of an instance of data. The input of a classification algorithm is a set of labeled examples. Each example is represented as a feature vector, and each label is an integer between 0 and k-1, where k is the number of classes. The output of a classification algorithm is a classifier, which can be used to predict the label of a new unlabeled instance. Examples of multi-class classification scenarios include:
+A supervised machine learning task that is used to predict the class (category) of an instance of data. The input of a classification algorithm is a set of labeled examples. Each label is an integer between 0 and k-1, where k is the number of classes. The output of a classification algorithm is a classifier, which you can use to predict the label of new unlabeled instances. Examples of multi-class classification scenarios include:
```

Comment:
Is it important to mention if the task is either supervised or unsupervised? How does it help a reader?

Also, in the last sentence it should be *...to predict the class...*

Reply:
I think supervised/unsupervised is important! These will become links when we add the definition into the Machine Learning Basics document.

Reply:
OK! By the way, for now, consider linking the related glossary sections:
https://docs.microsoft.com/en-us/dotnet/machine-learning/resources/glossary#supervised-machine-learning
for example

Reply:
That looks that glossary will grow out of one page into several ones. Last time I've updated it, I though about separate topics for regression and classifications. Above that, supervised and unsupervised will also become separate topics, most probably.

Reply:
Agreed. It's a good placeholder for now while we're still building out all of the content, but it can definitely split up into different pages in the future.